xblockutils/studio_editable_test.py

```diff
@@ -0,0 +1,54 @@
+from selenium.webdriver.support.ui import WebDriverWait
+from xblockutils.base_test import SeleniumXBlockTest
+
+
+class StudioEditableBaseTest(SeleniumXBlockTest):
```

Comment:
I believe that XBlock or the SDK now has bok_choy available and this is the preferable method of testing now. It's likely to come up that we're using Selenium directly here.